src/js/react/apps/health-station-search/components/ResultsMap.tsx

```diff
@@ -0,0 +1,77 @@
+import bucketToMap from '@/react/common/helpers/Aggregations';
```

Comment:
ResultsMap component could also be moved under Common components folder because it's also used in schools search and in the future can be used in other searches too.